### PR TITLE
[Smoke Test][rocPRIM] Updated smoke test to have shorter runtime and not use more than 2gb of vram

### PR DESCRIPTION
**Pull request by NguyenNhuDi**

### CODE REVIEW

**Comment by NguyenNhuDi** [comment]
This is a continuation for https://github.com/ROCm/rocPRIM/pull/629. Do not merge this PR until (#629) is merged.

File: rtest.py
Comment (umfranzw) [inline rtest.py:284]: I think this change looks great, but I just wanted to double-check something. It looks like on Windows, OS_info["VRAM"] is populated with the total amount of global memory on the device, not the amount that's currently free (see line 57). This means that the condition on line 284 will cause tests to be skipped only if the total amount of global memory on the device is less than 2GB. If that's the behaviour we're looking for, then I think this is fine. But if we're looking to skip the test if the amount of free memory is < 2GB, then we'll need to grab that info from a different line in the hipinfo output (the one labelled "meminfo.free").

Comment (NguyenNhuDi) [reply rtest.py:284]: I think the intent was to skip tests if global memory is less than 2GB

Comment (umfranzw) [reply rtest.py:284]: Ok, sounds good, thanks.

**Review by umfranzw — APPROVED** [review]
Looks good to me.

**Pull request merged**
